modules/installation-guide/partials/proc_configuring-workspace-exposure-strategies-using-an-operator.adoc

```diff
@@ -55,18 +57,21 @@ spec:
 ----
 
 
-
 * Update the YAML file properties of an already running cluster by executing the `{orch-cli} patch` command. For example:
 +
 [subs="+quotes,+attributes"]
 ----
 $ {orch-cli} patch checluster {prod-checluster} --type=json -p '[{"op": "replace", "path": "/spec/server/serverExposureStrategy", "value": "__single-host__"}] -n __<namespace-name>__'
 ----
-
-Depending on the strategy used, replace the `_single-host_` option in the above example with `multi-host` or `default-host`.
-
++
+Depending on the strategy used, replace the `_single-host_` option in the above example with `multi-host`
```

Comment:
This sentence doesn't make sense when `"{project-context}" != "che"`.

Reply:
Hello Robert. Thanks for asking. The single-host mode is one of the new features for the CRW 2.5 release, thus it makes sense also for the downstream docs and ifeval is no more needed.